src/index.ts

```diff
@@ -153,6 +153,18 @@ class Monika extends Command {
       description: 'Run Monika using a HAR file',
       multiple: false,
     }),
+
+    output: flags.string({
+      char: 'o', // (o)utput file to write config to
+      description: 'Write monika config file to this file',
+      multiple: false,
+    }),
+
+    postman: flags.string({
+      char: 'p', // (p)ostman
+      description: 'Run Monika using a Postman json file.',
+      multiple: false,
```

Comment:
we need to add `exclusive` property to prevent this flag from being used with conflicting flags such as `--config`. Check about the exclusive property [here](https://oclif.io/docs/flags)

Reply:
done